pkgs/build-support/setup-hooks/multiple-outputs.sh

```diff
@@ -138,7 +138,7 @@ moveToOutput() {
 
             # remove empty directories, printing iff at least one gets removed
             local srcParent="$(readlink -m "$srcPath/..")"
-            if rmdir "$srcParent"; then
+            if rmdir "$srcParent" 2>/dev/null; then
```

Comment:
Blanket-hiding errors is a bit worrying if the errors are not related to `-ENOENT`. My suggestion would be to test for directory presence: via `[[ -e $srcParent ]]` or `[[ -d $srcParent ]]` whichever is more appropriate.

Reply:
Ah, it runs on usually-non-empty dirs. Please ignore.

Reply:
maybe `if [[ -d path/to/dir && -n "$(ls -A path/to/dir)" ]]; then ` ? 
https://superuser.com/a/1332811